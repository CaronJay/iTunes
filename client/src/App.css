.apple{
  height:100px;
}


.myButton{
  background-color:black ;
  color: antiquewhite;

}

body {
  background-image: url("/background.jpg");
  background-attachment: fixed;
  color: black;
  flex:1;
  resize:contain;
  text-align: center;
  align-self:stretch; 
  text-decoration: none;
}
.album-container img:hover {
  transform: translateY(10px);
  overflow: hidden;
  overflow-anchor: none;
  text-decoration: none;
}

.album-container img {
  transition: 0.1s;
  text-decoration: none;
}

