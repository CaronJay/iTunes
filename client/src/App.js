import React,{Component} from 'react';
import apple from './apple.jpg';
import './App.css';
import { Albums } from './components/albums';
import NavBar from './components/Navbar';

class App extends Component {

  render() {
      return (
        <div className="App">
          {/*  */}
          <img className='apple' src = {apple} alt ="logo-apple"/>
          <div >
                <NavBar /><br></br>
            </div>
          
          <Albums/>
        </div>
      );
    }
}

export default App;
