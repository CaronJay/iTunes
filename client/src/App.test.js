import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer'
import Favourites from './components/Favourites'
import albums from './components/albums';


it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});


it('favourite added', () => {
  const tree = renderer
    .create(<saveFavourite></saveFavourite>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('grab query', () => {
  const tree = renderer
    .create(<albums></albums>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});