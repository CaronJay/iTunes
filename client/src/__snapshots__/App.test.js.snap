// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`favourite added 1`] = `<saveFavourite />`;

exports[`grab query 1`] = `<albums />`;
